style.css

```diff
@@ -21,26 +21,42 @@ nav a {
   margin-right: 1.5rem;
 }
 
+header ul {
```

Comment:
- [ ] Kindly check your mobile version and try to give the overall container a padding so that your elements can be centered and spaced correctly

![image](https://user-images.githubusercontent.com/98453979/194706501-29a59f6d-559f-4096-a8f7-a38a278600d0.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/194706625-35aae704-ec3c-4a47-a3c6-59d542295371.png)